automatic/tor-browser/update.ps1

```diff
@@ -23,14 +23,14 @@ function global:au_GetLatest {
   $download_page    = Invoke-WebRequest -Uri $releases -UseBasicParsing
     
   $allExes          = $download_page.Links | Where-Object href -match "\.exe$" | Select-Object -expand href
-  $url32            = $allExes | Where-Object { $_ -match "torbrowser-install-\d.*_All.exe$" } | Select-Object -First 1
-  $url64            = $allExes | Where-Object { $_ -match "torbrowser-install-win64.*_All.exe$" } | Select-Object -First 1
-  $version          = $url64 -split '\/' | Select-Object -last 1 -skip 1
+  $url32            = $releases + ($allExes | Where-Object { $_ -match "tor-browser-windows-x86_64-portable-\d.*.exe$" } | Select-Object -First 1)
+  $url64            = $releases + ($allExes | Where-Object { $_ -match "tor-browser-windows-x86_64-portable-\d.*.exe$" } | Select-Object -First 1)
+  $version          = $releases -split '\/' | Select-Object -last 1 -skip 1
```

Comment:
again, `$releases` should not contain a version number. Keep this line as it was.